src/aks-preview/azcli-aks-live-test/az_aks_tool/utils.py

```diff
@@ -0,0 +1,76 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
unit tests.

Reply:
updated in commit [f2fe148](https://github.com/Azure/azure-cli-extensions/pull/3292/commits/f2fe1484b1f5ecdafccdb2151169d226899d24aa).